dashboard/app/views/devise/registrations/_old_sign_up_form.html.haml

```diff
@@ -1,6 +1,6 @@
 - require 'geocoder'
 - require 'country_codes'
-- resource.user_type ||= params[:user].try(:fetch, :user_type) || User::TYPE_STUDENT
+- resource.user_type ||= params[:user].try(:fetch, :user_type, User::TYPE_STUDENT) || User::TYPE_STUDENT
```

Comment:
Does `try` take three parameters?  What prompted this change?

Reply:
`try` takes an arbitrary number of parameters; parameters after the first are passed to the function indicated by the first.

In this case, this means that `resource.user_type` will default to `User::TYPE_STUDENT` both if `params[:user].fetch` is undefined (the original case) and if `params[:user].fetch(:user_type)` returns nil.

This is specifically motivated by the fact that this partial will now be loaded even when we're in the "new form" case

Reply:
Ah, that makes sense.  Thanks.